### PR TITLE
Adds lots of New Items to Syndicate Kits

### PR DESCRIPTION
**Pull request by wesoda25**
## About The Pull Request

Updates recon, stealth, hacker, bond, sabotage, mad scientist, bee, and mr freeze bundles with several new items (most already in code) to make them less trash. 

## Why It's Good For The Game

No one really buys kits. Sure, they're fun once in a blue moon, basically when you can't come up with what to buy. But something that discourages many, or at least myself and several I've talked to, is the fact that you just get boring gear that makes greentext difficult. I've added some thematic new items to lots of them, to hopefully respark interest and make them a bit more deserving of their 20 TC purchase. 

## Changelog
:cl:
add: Updates recon, stealth, hacker, bond, sabotage, mad scientist, bee, and mr freeze bundles with several new items (most already in code) to make them less trash. 
add: Bee Sword (literally just a yellow, weaker esword variant, needed it for another piece of code)
add: A weaker variant of the decloner gun. 
/:cl:

Is my changelog good or do I have to specifically track each and every addition?

### CODE REVIEW

**Comment by wesoda25** [comment]
merge conflict always baffles me, works fine on private server and the commit in question seems fine to me

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -157,15 +164,19 @@
 			new /obj/item/assembly/signaler(src) // 0 tc
 			new /obj/item/assembly/signaler(src) // 0 tc
 			new /obj/item/storage/toolbox/syndicate(src) // 1 tc
-
+			new /obj/item/pen/edagger(src)
+			new /obj/item/gun/energy/wormhole_projector(src) //mooorttyyyy
```
Comment (Tlaltecuhtli) [inline code/game/objects/items/storage/uplink_kits.dm]: do we need this reference

File: code/game/objects/items/melee/energy.dm [hunk 2]
```diff
@@ -192,6 +192,16 @@
 	else
 		return ..()
 
+/obj/item/melee/transforming/energy/sword/bee  //yeah its fucking stupid but I wanted a yellow esword which is weaker than what we have
+	name = "Bee Sword"
```
Comment (Tlaltecuhtli) [inline code/game/objects/items/melee/energy.dm]: the stinger

Comment (wesoda25) [reply code/game/objects/items/melee/energy.dm]: Look i hate the fact that i had to code this already please dont make me revisit that file

File: code/game/objects/items/storage/uplink_kits.dm (hunk 1 above)
Comment (wesoda25) [reply code/game/objects/items/storage/uplink_kits.dm]: I can make the portals green if you like

Comment (wesoda25) [reply code/game/objects/items/storage/uplink_kits.dm]: Regardless its “sciencey” and its mad scientist bundle and honesty its such a shitty bundle even with these changes im wondering if people will still hate it, im just trying to buff it while staying with the theme

File: code/game/objects/items/melee/energy.dm (hunk 2 above)
Comment (Tlaltecuhtli) [reply code/game/objects/items/melee/energy.dm]: i guess i ll have to make it cool myself duhhhhhhhh also module filling isnt coding

Comment (wesoda25) [reply code/game/objects/items/melee/energy.dm]: if there are slashes and lots of lines it’s coding idiot read a book

**Comment by optimumtact** [comment]
power crepe

**Comment by Krysonism** [comment]
As someone who loves bundles i think this would be a great change. 

I am actually a big fan of the proper decloner(despite it being outshined by the ebow)

However is the 5 damage, 25 irradiate decloner realistically worth using in a firefight? I doubt it would. Bump it up to 10, 50 and i think it will be worthwhile.

**Comment by wesoda25** [comment]
> As someone who loves bundles i think this would be a great change.
> 
> I am actually a big fan of the proper decloner(despite it being outshined by the ebow)
> 
> However is the 5 damage, 25 irradiate decloner realistically worth using in a firefight? I doubt it would. Bump it up to 10, 50 and i think it will be worthwhile.

I’m considering it, but I’m worried the fact that it does clone damage, much harder to heal than other types, could be abused and make the bundle overpowered. For now I guess I will bump it up to 10, and if any balance issues present themselves we can revise at a later date.

**Comment by py01** [comment]
Is it really power creep if you’re improving items that are vastly below the power curve for traitor items?

**Comment by wesoda25** [comment]
I agree with py, bundles in their current state are really lame and you're infinitely better off just buying a surplus. Trying to round things out.

**Comment by ExcessiveUseOfCobblestone** [comment]
honestly it looks like they're just scaled up for better murderboning and I don't think that was the intention of bundles.

**Comment by wesoda25** [comment]
> honestly it looks like they're just scaled up for better murderboning and I don't think that was the intention of bundles.

You could argue any buff to anything traitor related makes them better at murderboning. The intention of bundles was to give you gear in order to complete your objectives in a gimmicky way, however the items given for most of the "gimmicks" are inadequate and boring. I'm just trying to make these things actually worth buying, because its honestly a waste of 20 TC. The real murdery ones I didn't even touch with this PR too, and most of my additions are adding little things to fit the theme (EMP flashlight for stealth, damp rag for bond, etc). If you want I will go through each and every addition and explain why I did it.

**Comment by Shadowflame909** [comment]
I mean for an example look at the carp hardsuit

IT LITERALLY HAS NEGATIVE ARMOR. Meaning you take more damage when you wear it.

Items might as well be orderable from cargo.

File: code/game/objects/items/storage/uplink_kits.dm [hunk 3]
```diff
@@ -10,7 +10,8 @@
 			new /obj/item/encryptionkey/syndicate(src) // 2 tc
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/uplink_kits.dm]: // 30 tc

File: code/game/objects/items/storage/uplink_kits.dm [hunk 4]
```diff
@@ -10,7 +10,8 @@
...
 			new /obj/item/storage/box/syndie_kit/space(src) //4 tc
 			new /obj/item/grenade/syndieminibomb/concussion/frag(src) // ~2 tc each?
-			new /obj/item/grenade/syndieminibomb/concussion/frag(src)
+			new /obj/item/grenade/syndieminibomb/concussion/frag(src) 
+			new /obj/item/flashlight/emp(src)
 
 		if("bloodyspai") // 27 tc now this is more right
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/uplink_kits.dm]: // 34 tc

File: code/game/objects/items/storage/uplink_kits.dm [hunk 5]
```diff
@@ -33,6 +34,8 @@
 			new /obj/item/chameleon(src)
 			new /obj/item/soap/syndie(src)
 			new /obj/item/clothing/glasses/thermal/syndi(src)
+			new /obj/item/flashlight/emp(src)
+			new /obj/item/jammer(src)
 
 		if("guns") // 28 tc now
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/uplink_kits.dm]: // 29 tc

File: code/game/objects/items/storage/uplink_kits.dm [hunk 6]
```diff
@@ -74,12 +77,14 @@
 			new /obj/item/encryptionkey/binary(src)
 			new /obj/item/aiModule/toyAI(src)
 			new /obj/item/multitool/ai_detect(src)
+			new /obj/item/storage/toolbox/syndicate(src)
 
 		if("lordsingulo") // 24 tc
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/uplink_kits.dm]: // 25 tc

File: code/game/objects/items/storage/uplink_kits.dm [hunk 7]
```diff
@@ -74,12 +77,14 @@
...
 			new /obj/item/sbeacondrop(src)
 			new /obj/item/clothing/suit/space/syndicate/black/red(src)
 			new /obj/item/clothing/head/helmet/space/syndicate/black/red(src)
 			new /obj/item/card/emag(src)
+			new /obj/item/storage/toolbox/syndicate(src)
 
 		if("sabotage") // 26 tc now
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/uplink_kits.dm]: // 28tc

File: code/game/objects/items/storage/uplink_kits.dm [hunk 8]
```diff
@@ -120,7 +126,8 @@
 			new /obj/item/clothing/under/chameleon(src)
 			new /obj/item/card/id/syndicate(src)
 			new /obj/item/reagent_containers/syringe/stimulants(src)
-
+			new /obj/item/reagent_containers/glass/rag(src)
+			
 		if("ninja") // 33 tc worth
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/uplink_kits.dm]: // 40 tc
Reason being warmhole projector feeling like a maybe 5tc? IDK on that

File: code/game/objects/items/storage/uplink_kits.dm [hunk 9]
```diff
@@ -157,15 +164,19 @@
 			new /obj/item/assembly/signaler(src) // 0 tc
 			new /obj/item/assembly/signaler(src) // 0 tc
 			new /obj/item/storage/toolbox/syndicate(src) // 1 tc
-
+			new /obj/item/pen/edagger(src)
+			new /obj/item/gun/energy/wormhole_projector(src) //mooorttyyyy
+			new /obj/item/gun/energy/decloner/unrestricted(src)
+			
 		if("bee") // ~25 tc
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/uplink_kits.dm]: // ~35 tc

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
Please update the notes on how much tc is gesamated in each pack.

File: code/game/objects/items/storage/uplink_kits.dm (hunk 3 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/storage/uplink_kits.dm]: I was unable to place this commit were the note is

**Comment by wesoda25** [comment]
Who cares its pointless, half of them don’t have them, half of them need to be changed due to item repricing, I’d argue TC count is only a good metric for showing that its under 50 TC, and for the special items I can scale it to whatever the fuck I want it at.

**Comment by necromanceranne** [comment]
The Bond bundle might actually need a more serious rework just because what stands out to me for the bond theme is his style, his gun and his gadgets that change every movie.

The bundle needs random gadgets. Maybe even unique ones to the kit itself and some that are already in the uplink since many of them draw inspiration from actual bond gadgets/irl spy gear.

**Comment by Akrilla** [comment]
> Who cares its pointless, half of them don’t have them, half of them need to be changed due to item repricing, I’d argue TC count is only a good metric for showing that its under 50 TC, and for the special items I can scale it to whatever the fuck I want it at.

Yeah tbh I don't really know what they bothered add those TC comments. It's not really ever going to be properly maintained.

**Comment by Tlaltecuhtli** [comment]
the hacker module should be changed, you can already buy the whole kit with tcs

**Comment by wesoda25** [comment]
> the hacker module should be changed, you can already buy the whole kit with tcs

Gave it some more items that I hope fit the theme (useful at least for avoiding capture when hacking was my reasoning). 

Also got rid of TC headings they're pointless anyways, kept some things tho.

**Pull request merged**